analysis.go

```diff
@@ -155,9 +156,18 @@ func ListPackages(fileRoot, importRoot string) (PackageTree, error) {
 		ip := filepath.ToSlash(filepath.Join(importRoot, strings.TrimPrefix(path, fileRoot)))
 
 		// Find all the imports, across all os/arch combos
-		p, err := ctx.ImportDir(path, analysisImportMode())
+		ap, err := filepath.Abs(path)
```

Comment:
curious what your reasoning was about adding this - is there something later relying on an absolute path?